README.md

```diff
@@ -55,7 +55,7 @@ BET | [komodo](https://github.com/komodoplatform/komodo) | [e159b4e](https://git
 BOTS | [komodo](https://github.com/komodoplatform/komodo) | [e159b4e](https://github.com/KomodoPlatform/komodo/tree/e159b4e7a40d3886519401c4074e957a1f9d42ba) | [![dPOW Status](http://116.203.64.110/svg/BOTS_badge.svg?maxAge=60)](https://komodostats.com) | dPoW-mainnet
 BTCH | [komodo](https://github.com/komodoplatform/komodo) | [e159b4e](https://github.com/KomodoPlatform/komodo/tree/e159b4e7a40d3886519401c4074e957a1f9d42ba) | [![dPOW Status](http://116.203.64.110/svg/BTCH_badge.svg?maxAge=60)](https://komodostats.com) | dPoW-mainnet
 CCL | [komodo](https://github.com/komodoplatform/komodo) | [e159b4e](https://github.com/KomodoPlatform/komodo/tree/e159b4e7a40d3886519401c4074e957a1f9d42ba) | [![dPOW Status](http://116.203.64.110/svg/CCL_badge.svg?maxAge=60)](https://komodostats.com) | dPoW-mainnet
-CHIPS | [chips](https://github.com/jl777/chips3/tree/master) | [31d59f9](https://github.com/jl777/chips3/tree/31d59f9d8fa4a8e00dd474ef0561a5b174056d86) | [![dPOW Status](http://116.203.64.110/svg/CHIPS_badge.svg?maxAge=60)](https://komodostats.com) | dPoW-3P
+CHIPS | [chips](https://github.com/jl777/chips3/tree/master) | [2ce1c38](https://github.com/jl777/chips3/tree/2ce1c3834a3f58ecff77a4552c3a789011d653eb) | [![dPOW Status](http://116.203.64.110/svg/CHIPS_badge.svg?maxAge=60)](https://komodostats.com) | dPoW-3P
```

Comment:
chips upstream is https://github.com/chips-blockchain/chips atm
https://github.com/KomodoPlatform/dPoW/pull/194/files#diff-483788a18af34255299a870b35f2fccdR16

and aPoW merged into it: https://github.com/chips-blockchain/chips/pull/2

Reply:
we can update it to our self hosted repo with the next dpow update - security review was based on the repositories referred in https://github.com/KomodoPlatform/dPoW/pull/193 so this is fine - i.e. https://github.com/KomodoPlatform/dPoW/pull/193#pullrequestreview-474878565